setup.py

```diff
@@ -87,4 +90,15 @@
         "Operating System :: OS Independent",
         "Topic :: Internet :: WWW/HTTP",
     ],
+    # Additional functionality
+    # Scripts to include in the distribution
+    scripts=["scripts/googleapiclient_cli.py"],
+    # Additional data files to include in the distribution
+    data_files=[("config", ["config/googleapiclient.cfg"])],
+    # Additional entry points for command line scripts
+    entry_points={
+        "console_scripts": [
+            "googleapiclient_cli = googleapiclient.scripts.googleapiclient_cli:main"
```

Comment:
Hi @aria1991,
The module `googleapiclient_cli` does not exist [here](https://github.com/googleapis/google-api-python-client/tree/main/scripts). Please could you take a look?

Reply:
Hi @parthea thanks for your comment. My bad! I added to the `keywords` in my new commit, you can ignore it. Do you have any suggestion on adding it (sth like requirement text file) otherwise we can ignore this part and remove it!